### PR TITLE
[CIRCLE-6546] Ensure that each AWS Region has the Ubuntu 14.04 AMI that we want to use.

### PR DESCRIPTION
**Pull request by MikeyYeahYeah**

### CODE REVIEW

File: templates/services_user_data.tpl [hunk 1]
```diff
@@ -1,18 +1,39 @@
 #!/bin/bash
 
-export PRIVATE_IP="$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')"
+REPLICATED_VERSION="2.10.3"
+
+echo "-------------------------------------------"
+echo "     Performing System Updates"
+echo "-------------------------------------------"
+apt-get update and apt-get -y upgrade
```
Comment (MikeyYeahYeah) [inline templates/services_user_data.tpl]: Do we have duplicated  updates now?

File: templates/output.tpl [hunk 2]
```diff
@@ -13,4 +13,4 @@ ${ ansible ? "To rerun the Ansible provisioner using your `${ssh_key}` private k
 
     ansible-playbook playbook.yml -v -i ./.ansible/hosts -e '@./.ansible/extra_vars.json'" : "" }
 
-Thank you and enjoy using CircleCI Enterprise!
+Thank you and enjoy using CircleCI Server!
```
Comment (MikeyYeahYeah) [inline templates/output.tpl]: Given Nathan'c comments in #release we should probably remove Server and just have CircleCI... cuz... you know... bizdev....

Comment (levlaz) [reply templates/output.tpl]: I thought that we were explicitly supposed to say "Server". Im so confused lol. @ndintenfass ?

File: templates/services_user_data.tpl (hunk 1 above)
Comment (levlaz) [reply templates/services_user_data.tpl]: Oops. Yeah.

File: templates/output.tpl (hunk 2 above)
Comment (levlaz) [reply templates/output.tpl]: I just read his thing. that motivates me. Will change.

File: templates/builders_user_data.tpl [hunk 3]
```diff
@@ -28,11 +28,21 @@ echo "      Pulling Server Builder Image"
...
 
+# Make a new docker network
+docker network create -d bridge -o "com.docker.network.bridge.name"="circle0" circle-bridge
+
+# Block traffic from build containers to the EC2 metadata API
+iptables -I FORWARD -d 169.254.169.254 -p tcp -i docker0 -j DROP
+
+# Block traffic from build containers to non-whitelisted ports on the services box
+iptables -I FORWARD -d $SERVICES_PRIVATE_IP -p tcp -i docker0 -m multiport ! --dports 80,443 -j DROP
+
```
Comment (MikeyYeahYeah) [inline templates/builders_user_data.tpl]:
```
> make docker the default choice for AWS, but that we should also block access to 169.254.169.254 from builds if we do that
```
Can we explain this a little more?  What is meant by make Docker default choice?  Why are we blocking that IP address?

Comment (MikeyYeahYeah) [reply templates/builders_user_data.tpl]: Also... have we tested this?

**Pull request closed without merging**